js/internal/modules/modules.go

```diff
@@ -61,3 +61,25 @@ func Register(name string, mod interface{}) {
 	}
```

Comment:
The check above here is done only against the modules that are *registered* through the *Register* call. 

While the current usage is such they are required to have `x/` in the public `modules` module, it still isn't *technically* correct.

So should I:
1. do nothing
2. drop the internal `modules` package and move everything to the public one. This way I will check it there and hope that nobody will add `x/k6` standard lib.
3. Actually check against the result of `GetJSModules`

Reply:
Hhmm considering we only made the package internal to begin with was to distinguish between internal and external module registration, and internal modules don't call `Register()` anymore, I vote for the 2nd approach, and to do any additional checks in the public package.

Reply:
Not sure I grok everything fully, but yeah, 2nd approach seems best :+1:, we don't need an `internal` module anymore.